src/gui/pages_game/KM_GUIGameHouse.pas

```diff
@@ -876,8 +888,8 @@ procedure TKMGUIGameHouse.House_SchoolUnitChange(Sender: TObject; Shift: TShiftS
 end;
 
 
-{Process click on Remove-from-queue buttons of School}
-procedure TKMGUIGameHouse.House_SchoolUnitRemove(Sender: TObject; Shift: TShiftState);
+{Process click on Units bar buttons of School}

```

Comment:
Please change into:
// Process click on Units queue buttons of School

as mentioned above, there's no unit "bar"